.github/workflows/plugin-assets.yml

```diff
@@ -0,0 +1,16 @@
+name: Plugin assets/readme update
+on:
+  push:
+    branches:
+      - master
```

Comment:
@gonzomir this action will only run on every push to master. If you want it to make it run exclusively when a specific file is changed, please try something like:

```yaml
on:
  push:
    branches:
      - master
    paths:
      - 'readme.txt'
      - 'assets/**'
      - 'etc'

```